packages/protocol-utils/src/transformer_utils.ts

```diff
@@ -135,6 +135,7 @@ export enum BridgeProtocol {
     AaveV2,
     Compound,
     BalancerV2Batch,
+    BiSwap,
```

Comment:
```suggestion
    BiSwap,
```
This isn't needed, as it has a 1:1 mapping to "custom bridge protocol" implementations inside of the [BridgeAdapter contract](https://github.com/0xProject/protocol/blob/development/contracts/zero-ex/contracts/src/transformers/bridges/BridgeProtocols.sol#L55). Since this is just a UniswapV2? clone, there is no need to update [BridgeAdapter](https://github.com/0xProject/protocol/blob/development/contracts/zero-ex/contracts/src/transformers/bridges/BridgeProtocols.sol#L55) and then no need to map it here.

Reply:
yeah i think this is the things causing asset-swapper to fail on build with balancerV2Batch. other than that lgtm with a simbot